### PR TITLE
List: Fix "Select All" aria-label localization (T1239880)

### PR DESCRIPTION
**Pull request by jdvictoria**

### CODE REVIEW

**Comment by jdvictoria** [comment]
[List - "Select All" aria-label is not localized [T1239880]](https://trello.com/c/1y53C5nH/9035-list-select-all-aria-label-is-not-localized-t1239880)

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/list.markup.tests.js [hunk 1]
```diff
@@ -524,13 +527,36 @@ QUnit.module('decorators markup', {}, () => {
...
+        const $multipleContainer = $list.find(toSelector(SELECT_ALL_CLASS));
         assert.equal($multipleContainer.length, 1, 'container for SelectAll rendered');
         assert.equal($multipleContainer.text(), 'Test', 'select all rendered');
         const $checkbox = $multipleContainer.find('.dx-checkbox');
         assert.equal($checkbox.length, 1, 'checkbox rendered');
     });
 
+    QUnit.test('selectAll text and aria-label attribute should be equal to custom localized text (T1239880)', function(assert) {
+        localization.loadMessages({ 'en': { 'dxList-selectAll': 'custom-select-all' } });
+        localization.locale('en');
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/list.markup.tests.js]: `en` is a default locale, so no need to do it
```suggestion
```

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/list.markup.tests.js [hunk 2]
```diff
@@ -524,13 +527,36 @@ QUnit.module('decorators markup', {}, () => {
...
-        const $multipleContainer = $list.find('.dx-list-select-all');
+        const $multipleContainer = $list.find(toSelector(SELECT_ALL_CLASS));
         assert.equal($multipleContainer.length, 1, 'container for SelectAll rendered');
         assert.equal($multipleContainer.text(), 'Test', 'select all rendered');
         const $checkbox = $multipleContainer.find('.dx-checkbox');
         assert.equal($checkbox.length, 1, 'checkbox rendered');
     });
 
+    QUnit.test('selectAll text and aria-label attribute should be equal to custom localized text (T1239880)', function(assert) {
+        localization.loadMessages({ 'en': { 'dxList-selectAll': 'custom-select-all' } });
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/list.markup.tests.js]: The test asserts will be a bit more intuitive of we store this value to a variable.
```javascript
const localizedSelectAllText = 'custom-select-all';
localization.loadMessages({ 'en': { 'dxList-selectAll': localizedSelectAllText } });
// ...

assert.equal($selectAllCheckBox.attr('aria-label'), localizedSelectAllText, 'selectAll checkbox aria-label should be equal to localized text');
```

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/list.markup.tests.js [hunk 3]
```diff
@@ -524,13 +527,36 @@ QUnit.module('decorators markup', {}, () => {
...
+                { id: 3, text: 'Item 3' },
+            ],
+            showSelectionControls: true,
+            selectionMode: 'all',
+        }));
+
+        const $selectAllCheckBox = $list.find(toSelector(SELECT_ALL_CHECKBOX_CLASS));
+        const $multipleContainer = $list.find(toSelector(SELECT_ALL_CLASS));
+
+        assert.equal($selectAllCheckBox.attr('aria-label'), 'custom-select-all', 'selectAll checkbox aria-label should be equal to localized text');
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/list.markup.tests.js]: Actually no diff in this case, but it's better to not break the habit
```suggestion
        assert.strictEqual($selectAllCheckBox.attr('aria-label'), 'custom-select-all', 'selectAll checkbox aria-label should be equal to localized text');
```

**Review by ksercs — DISMISSED** [review]

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/list.markup.tests.js (hunk 1 above)
Comment (jdvictoria) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets/list.markup.tests.js]: [b71911d](https://github.com/DevExpress/DevExtreme/pull/27664/commits/b71911da4022244261cdc0aa630106f28be9cee3)

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/list.markup.tests.js (hunk 2 above)
Comment (jdvictoria) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets/list.markup.tests.js]: [c38c1ec](https://github.com/DevExpress/DevExtreme/pull/27664/commits/c38c1ecfd87f6a8d044e3a867871ffbfa7f3fa25)

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/list.markup.tests.js (hunk 3 above)
Comment (jdvictoria) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets/list.markup.tests.js]: [b901b96](https://github.com/DevExpress/DevExtreme/pull/27664/commits/b901b967eb648ba2c37c60a7dcfab427de681ee8)

**Review by ksercs — APPROVED** [review]

**Review by AlexanderMoiseev — APPROVED** [review]

**Comment by jdvictoria** [comment]
[List - "Select All" aria-label is not localized [T1239880]](https://trello.com/c/1y53C5nH/9035-list-select-all-aria-label-is-not-localized-t1239880)

**Pull request merged**